doc/code-generator/BUILDME.md

```diff
@@ -0,0 +1,84 @@
+<a name="top" />
+
+TERMHUB REGENERATE CLIENT SDKs
+==================================
+This is a how-to guide for rebuilding openapi generated code as updates to the swagger are implemented.
+
+## Requirements
+
+1. Java 17+
```

Comment:
Do we really need gradle installed? Just running ./gradlew should bring in the required gradle

Reply:
Valid, I can remove that